store/views.py

```diff
@@ -77,6 +145,64 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    
+    bookcopy_id = request.POST.get('bid')
+    bookcopy = get_object_or_404(BookCopy,id=bookcopy_id)
+    bookcopy.borrower = None
+    bookcopy.borrow_date = None
+    bookcopy.status = True
+    bookcopy.save()
+    print(request.POST)
+    msg = 'success'
+
+    response_data={
+        'message': msg
+    }
+
+    return JsonResponse(response_data)
+
+def rateBook(request,bid):
+    if request.method == 'POST':
+        form = forms.RatingForm(request.POST)
+        if form.is_valid():
+            rating = form.cleaned_data.get('rating')
+            ratings_by_user =request.user.rating_by.all()
+            print(ratings_by_user,'hellllsaldlasldlasldl')
+            if ratings_by_user.count() > 0:
+                rating_object = ratings_by_user[0]
+                rating_object.rating = rating
+                rating_object.save()
```

Comment:
You've not put a backend validation on the rating, so the user can simply edit the JS code you've written in the template and easily put invalid values of rating.